### PR TITLE
Add --config option to the ssc init

### PR DESCRIPTION
**Pull request by chicoxyzzy**
This helps to wrap the existing application into Socket Runtime

### CODE REVIEW

**Comment by jwerle** [comment]
Maybe it's worthwhile just checking if files exist before over writing them?

**Comment by getify** [comment]
I agree with @jwerle here. I think the creation of the directory and files is helpful but should be guarded on whether they don't yet exist, rather than either a potential error OR (worse) overwriting them.

File: src/cli/cli.cc [hunk 1]
```diff
@@ -960,7 +960,7 @@ int main (const int argc, const char* argv[]) {
         }
 
         // default values
-        settings["build_output"] = settings["build_output"].size() > 0 ? settings["build_output"] : "dist";
+        settings["build_output"] = settings["build_output"].size() > 0 ? settings["build_output"] : "build";
```
Comment (chicoxyzzy) [inline src/cli/cli.cc:963]: so now we have `"build"` by default in both `.gitignore` and as an implicit default value

**Comment by chicoxyzzy** [comment]
made it exit early when `socket.ini` exists and skip creating `.gitignore` and `src/index.html` if they exist

**Review by getify — APPROVED** [review]
LGTM

**Review by jwerle — APPROVED** [review]

File: src/cli/templates.hh [hunk 2]
```diff
@@ -1379,14 +1382,15 @@ flags = -O3
...
+; A directory where your application's code is located. ssc will copy everything in this directory
+; to the build output directory. It's ignored if the "script" property is set.
 input = "src"
 
 ; The name of the program and executable to be output. Can't contain spaces or special characters. Required field.
 name = "beepboop"
 
 ; The binary output path. It's recommended to add this path to .gitignore.
-output = "dist"
+output = "build"
```
Comment (jwerle) [inline src/cli/templates.hh:1393]: finally, thank you

**Review by jwerle — APPROVED** [review]

**Pull request merged**